packages/icu4c.rb

```diff
@@ -11,10 +11,10 @@ class Icu4c < Package
   binary_compression 'tar.zst'
 
   binary_sha256({
-    aarch64: '2ff2ed4760529e3611a51211876c17df5c9acc5a689cd35cb5a98ce0c4c1f714',
-     armv7l: '2ff2ed4760529e3611a51211876c17df5c9acc5a689cd35cb5a98ce0c4c1f714',
-       i686: '3bab33be0214ced3080b895c615ee8f9bf3b98e191701be0917c2e02c783c26f',
-     x86_64: '83e81c0d8bb9981034d6d63755ac0e9ae833d9544cbc07e32d242e98cf3d30db'
+    aarch64: 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa',
+     armv7l: 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa',
+       i686: 'iiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii',
+     x86_64: 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx'
```

Comment:
How did this happen?

Reply:
Having those lines in there keeps crew from building a package twice...

Reply:
Which is another bug, I guess... 😅

Reply:
Well, you'll never get this merged until it returns back to the original non-broken version. :)  Why not create a special package for testing like broken_for_tests.rb?

Reply:
once i confirm the tests are working i'll just revert these changes, so no biggie

Reply:
In the meantime, I changed this to draft so it doesn't get inadvertently merged.

Reply:
As a general rule, we should convert all PRs to draft that are not ready to be merged.

Reply:
fair